CMakeLists.txt

```diff
@@ -23,6 +23,10 @@ if (NOT DEFINED Spglib_IS_TOP_LEVEL)
 	endif ()
 endif ()
 
+# Specify C standard
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_C_STANDARD_REQUIRED True)
```

Comment:
(Note for me) This is the first time for spglib to require C standard. We should mention it in the release note for the next version (2.1.0).

Reply:
Indeed. It is hard to tell if there was an implicit standard used before, but it is always good to be explicit.

Technically it is not strictly C11 because it covers pre C11 proposal of `_Thread_local`, but that is plenty of a safe standard to use these days. Heck, even C17 should be fine to implement now if there is anything we need there.